docs/products/grafana/howto/replace-expression-string.rst

```diff
@@ -0,0 +1,102 @@
+``YOUR_DASHBOARD_URL``     The URL for the Grafana dashboard
+----------------------     -------------------------------------------------------------
+``YOUR_DASHBOARD_UID``     The UID that identifies the Grafana dashboard
+----------------------     -------------------------------------------------------------
+``OLD_STRING``             The old string, that you want to replace
+----------------------     -------------------------------------------------------------
+``NEW_STRING``             The new string, that you want to use instead
+======================     =============================================================
+
+Gather authentication and authorization values
+----------------------------------------------
+
+To get your API key (``YOUR_API_KEY``):
+
+* Go to your Grafana® UI. Select the **Configuration** icon and then select the **API keys** tab
+
+* If you already have an appropriate API key available, save a copy of it.
+
+* Otherwise, select **Add API key** and fill in the *Key name*, *Role* and *Time to live*. Click **Add** and then save the new API key.
+
+   .. note:: *Role* must be either *Editor* or *Admin*.
+
+To get the dashboard URL and UID (``YOUR_DASHBOARD_URL`` and ``YOUR_DASHBOARD_URI``):
+
+* Go to the dashboard on which you want to change metric expression strings.
+
+* Save the URL.
+
+* Select the **Dashboard settings** icon, then select **JSON Model**. Save the dashboard *UID* from the *JSON Model* page.
+
+Perform the replacement
+-----------------------
+
+Use a command like the following to perform the replacement, changing the placeholder variable names to the values you collected above:
+
+.. code:: bash
+
+  aiven-string-replacer-for-grafana -apikey YOUR_API_KEY -url YOUR_DASHBOARD_URL \
+      -from OLD_STRING -to NEW_STRING -uid YOUR_DASHBOARD_UID
+
+Example: changing ``elasticsearch_`` to ``opensearch_``
+-------------------------------------------------------
+
+For instance, use the following command to change all metric expressions that start with ``elasticsearch_`` to instead start with ``opensearch_``:
+
+.. code:: bash
+
+  aiven-string-replacer-for-grafana -apikey YOUR_API_KEY -url YOUR_DASHBOARD_URL \
+      -from elasticsearch_ -to opensearch_ -uid YOUR_DASHBOARD_UID
+
+The change will be visible in the *Query* panel:
+
+* Before running the command, metrics start with ``elasticsearch_``:
+
+  .. image:: /images/products/grafana/query-with-elasticsearch-prefix.png
+      :alt: A screenshot of the Grafana Dashboard query showing metrics prefixed with ``elasticsearch_``
+
+* After running the command, metrics start with ``opensearch_``:
+
+  .. image:: /images/products/grafana/query-with-opensearch-prefix.png
+      :alt: A screenshot of the Grafana Dashboard query showing metrics prefixed with ``opensearch_``
+
+Use the version history to revert
+---------------------------------
+If necessary, the *Dashboard changelog* can be used to revert a change.
```

Comment:
Is that actually correct - that is, can one use the changelog page to revert a change, or is that just giving the version information needed to revert a change?

Reply:
Yep it is correct, I add a bit more text so it would be more descriptive.

If necessary, the *Dashboard changelog* page can be used to revert a change to a specific version.